setup.py

```diff
@@ -1,14 +1,15 @@
 """Setup.py for the lakeFS Airflow provider package"""
 
 from setuptools import find_packages, setup
+from lakefs_provider.version import __version__
```

Comment:
from what I understand it is not recommended to import the package from setup.py